.github/workflows/pylib-tests.yml

```diff
@@ -0,0 +1,30 @@
+name: Pip install test
```

Comment:
This file could be renamed to `python-package-tests.yaml` to make it consistent with the others.